hms-app/src/common/redux/hackathonSlice.ts

```diff
@@ -11,7 +11,8 @@ const initialState: HackathonState = {
   hackathons: [
```

Comment:
if this is the list of all hackathons, shouldn't it contain the next and previous ones as well?

Reply:
its just to show redux how a hackathon looks like